script/ethereum/DeployMorphoTokenEthereum.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.0;
+
+import "../../lib/forge-std/src/Script.sol";
+import "../../lib/forge-std/src/console.sol";
+
+import {MorphoTokenEthereum} from "../../src/MorphoTokenEthereum.sol";
+import {Wrapper} from "../../src/Wrapper.sol";
+import {ERC1967Proxy} from
+    "../../lib/openzeppelin-contracts-upgradeable/lib/openzeppelin-contracts/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+import {DeploymentImplementation} from "../helpers/DeploymentImplementation.sol";
+
+contract DeployMorphoTokenEthereum is Script {
+    address public constant MORPHO_DAO = 0xcBa28b38103307Ec8dA98377ffF9816C164f9AFa;
+
+    bytes32 public DEPLOYMENT_IMPLEMENTATION_SALT;
+    bytes32 public IMPLEMENTATION_SALT;
+    bytes32 public PROXY_SALT;
+    bytes32 public WRAPPER_SALT;
+
+    address constant DEPLOYER = 0x937Ce2d6c488b361825D2DB5e8A70e26d48afEd5;
```

Comment:
What is this address ? Maybe document and secure it, so we know it will be available for future deployments (to keep the same address of the token on different chains)